Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 11th Bug Crusade

## Changelog:
- Fixed Old Staroverstvo having no localization after a reformation.
- Ogre and Saurok provinces in 603 and 605 bookmarks having Fetishistic faith instead of their racial faith.
- Fixed Dark governments are not able to use castle buildings.
- Fixed spawned Grommash being born in 540s.
- Fixed unlocalized `can_have_sexual_relation_with_trigger_tooltip_first_not` appeared in conditions when you tried to marry wrong race.
- Fixed demon Eredar having no beards and hairs.
- Fixed Sylvanas not having her tears in the year 605 bookmark.
- Fixed not getting a notification when the spouse turns undead.
- Fixed the Alliance not creating after the Horde wins Stormwind.
- Fixed Kael'thas having a beard.
- Fixed Groundwizzle dynasty having Ogre race trait and being Gnoll culture.


## Developer changelog:
- Fixed `copy_inheritable_appearance_from` error due to the character being purged from game files.


## Tests:


# How to test:
Confirm bugfixes from the changelog works.

A few tips:
Right-click a character > Debug section > Become Undead to make a character undead.
Start in the year 583 bookmark, `event WCDRA.998` to spawn Grommash.
